webui/react/src/pages/ProjectDetails.tsx

```diff
@@ -108,7 +108,7 @@ const ProjectDetails: React.FC = () => {
   const [ total, setTotal ] = useState(0);
   const [ canceler ] = useState(new AbortController());
   const pageRef = useRef<HTMLElement>(null);
-  const { canDeleteExperiment, canMoveExperiment, canViewWorkspaces } = usePermissions();
+  const { canDeleteExperiment, canMoveExperiment } = usePermissions();
```

Comment:
question: Why was this removed? Seems unrelated to the rest of the PR

Reply:
oh I accidentally removed it. or its not used?